client/src/StudentHomePage/StudentHomePage.js

```diff
@@ -8,34 +16,93 @@ class StudentHomePage extends React.Component{
         super(props);
        
         this.state = {
-            info: [
-                {todo: "Grade", due_date: "2/1 Mon. 11:59 PM", class_name: "CST-399", assignment: "HW1"},
-                {todo: "Submit", due_date: "2/2 Tue. 11:59 PM", class_name: "CST-205", assignment: "HW2"},
-                {todo: "Grade", due_date: "2/3 Wed. 11:59 PM", class_name: "CST-340", assignment: "HW3"},
-                {todo: "Submit", due_date: "2/4 Thur. 11:59 PM", class_name: "CST-380", assignment: "HW4"},
-                {todo: "Grade", due_date: "2/5 Fri. 11:59 PM", class_name: "CST-399", assignment: "HW5"},
-            ]   
+            studentInfo: [{
+                class_id: 1,
+                professor_email: "",
+                name: "",
+            }],  
+            assignments: [
+                {
+                    rubric_id: 100,
+                    assigned_date: "2020-04-21T07:00:00.000Z ",
+                    due_date: "2020-04-28T07:00:00.000Z",
+                    final_due_date: "2020-05-01T07:00:00.000Z ",
+                    section_id: 1 
+                },
+                {
+                    rubric_id: 200,
+                    assigned_date: "2020-04-21T07:00:00.000Z ",
+                    due_date: "2020-04-28T07:00:00.000Z",
+                    final_due_date: "2020-05-01T07:00:00.000Z ",
+                    section_id: 1 
+                }
+            ],
+            isHovered: false 
         }
+        
+        this.toggleHover = this.toggleHover.bind(this);
+    }
+
+    toggleHover(){
+        this.setState(prevState => ({isHovered: !prevState.isHovered}));
+    }
+
+
+    componentDidMount(){
+    	axios({
+            method: 'get',
+            url:'https://rwlautograder.herokuapp.com/api/stud/registered/class/info'
```

Comment:
You are missing the header argument for axios, it should look something like this
```
    	axios({
            method: 'get',
            url:'https://rwlautograder.herokuapp.com/api/stud/registered/class/info',
            headers: {
                  'content-type': 'application/x-www-form-urlencoded;charset=utf-8',
                  'Authorization':  'Bearer asdf.asdf.asdf'
             }
      })
```